Endava.TechCourse.BankApp.Application/Commands/CreateTransactionByEmail/CreateTransactionByEmailHandler.cs

```diff
@@ -0,0 +1,61 @@
+using Endava.TechCourse.BankApp.Domain.Models;
+using Endava.TechCourse.BankApp.Infrastructure.Persistence;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace Endava.TechCourse.BankApp.Application.Commands.CreateTransactionByEmail;
+
+public class CreateTransactionByEmailHandler : IRequestHandler<CreateTransactionByEmailCommand, CommandStatus>
+{
+    private readonly ApplicationDbContext context;
+
+    public CreateTransactionByEmailHandler(ApplicationDbContext context)
+    {
+        ArgumentNullException.ThrowIfNull(context);
+
+        this.context = context;
+    }
+
+    public async Task<CommandStatus> Handle(CreateTransactionByEmailCommand request,
```

Comment:
You are doing a founds transfer here , a transaction is a side effect , so a better name for this command would be "TransferFoundsByEmail"